index.html

```diff
@@ -127,7 +127,6 @@
             'error', loadErrHandler, true /* capture phase type of listener */);
     </script>
     <script><!--#include virtual="/config.js" --></script><!-- adapt to your needs, i.e. set hosts and bosh path -->
-    <script src="static/utils.js?v=1"></script>
```

Comment:
Maybe I slightly still prefer the old implementation but I don't have strong arguments. 

I don't mind the current one since the size is smaller than the unminified old version.